ludwig/schema/features/preprocessing/image.py

```diff
@@ -37,7 +37,7 @@ class ImagePreprocessingConfig(BasePreprocessingConfig):
         parameter_metadata=FEATURE_METADATA[IMAGE][PREPROCESSING]["computed_fill_value"],
     )
 
-    height: int = schema_utils.PositiveInteger(
+    height: Optional[int] = schema_utils.PositiveInteger(
```

Comment:
It's not redundant, because the Optional here is primarily useful for static analysis when the config is being used in the Ludwig code. If it's possible for the value to be `None`, but the type is `int`, then the expectation from the API user should be that they can treat the value as an `int` without needing to do any checks for `None`. But this will lead to them getting `NoneType` exceptions. Does that make sense?

Reply:
+1. Even if it's redundant, should be marked with `Optional` or `Union[..., None]`